server/src/test/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActorSpec.scala

```diff
@@ -55,6 +56,13 @@ class MaterializeWorkflowDescriptorActorSpec extends CromwellTestKitWordSpec wit
   val callCachingEnabled = true
   val invalidateBadCacheResults = true
 
+  val validMemoryRetryOptions1 = WorkflowOptions.fromJsonString(""" { "memory_retry_multiplier": 1.0 } """).get
+  val validMemoryRetryOptions2 = WorkflowOptions.fromJsonString(""" { "memory_retry_multiplier": 99.0 } """).get
+  val validMemoryRetryOptions3 = WorkflowOptions.fromJsonString(""" { "memory_retry_multiplier": 12.34 } """).get
+  val invalidMemoryRetryOptions1 = WorkflowOptions.fromJsonString(""" { "memory_retry_multiplier": 0.9 } """).get
+  val invalidMemoryRetryOptions2 = WorkflowOptions.fromJsonString(""" { "memory_retry_multiplier": 99.1 } """).get
+  val invalidMemoryRetryOptions3 = WorkflowOptions.fromJsonString(""" { "memory_retry_multiplier": -1.1 } """).get
```

Comment:
This might be a good place to have an additional:

```
    val invalidMemoryRetryOptions4 = WorkflowOptions.fromJsonString(""" { "memory_retry_multiplier": "one point one" } """).get
```